app/javascript/components/widgets/weather/weather-effect.jsx

```diff
@@ -248,6 +248,52 @@ const Thunder = () => {
     </>
   );
 };
+// #endregion
+
+// cloud starts here
+const CloudMotionFrames = keyframes`
+   0% {
+    margin-left:-1000px;
+    
+  }
+  100% {
+    margin-left:100%;
+  }
+`;
+
+const Motion = styled.div`
+  bottom: 100%;
+  pointer-events: none;
+  position: absolute;
+  top: 50px;
+`;
+
+const MovingCloud = styled(Motion)`
+  opacity: 60%;
+  background: white;
+  width: 200px;
+  height: 150px;
+  border-radius: 0 0 50px 50px;
+  background: radial-gradient(35px 30px, white 98%, transparent 100%) 20% 30% /70px
+      60px,
+    radial-gradient(50px 45px, white 98%, transparent 100%) 50% 50% /100px 90px,
+    radial-gradient(50px, white 98%, transparent 100%) 100% 100%/100px 100px,
+    radial-gradient(40px, white 98%, transparent 100%) 0 100% /80px 80px,
+    linear-gradient(white, white) bottom/100% 40px;
+  background-repeat: no-repeat;
+  animation: ${CloudMotionFrames} linear;
+`;
+
+const Cloud = ({ duration, variance, count }) => (
+  <WeatherWrapper {...{ Type: MovingCloud, duration, variance, count }} />
```

Comment:
I think `WeatherWrapper` isn't a good fit here since its setting the `left` style property to distribute across the screen horizontal for vertical animation. We should create a new `HorizontalWeatherWrapper` that switches this, vary the vertical position of clouds (say some where between 75%-100% near the top of the city line, moving the clouds horizontally.